packages/universal-wallet-fastify-plugin/src/routes/base.ts

```diff
@@ -0,0 +1,69 @@
+export default (options: any) => {
+  return (fastify: any) => {
+    fastify.post(
+      `/:${options.walletId}/wallet/add`,
+      {
+        preValidation: options.hooks ? options.hooks.preValidation : [],
+      },
+      async (request: any, reply: any) => {
+        const wallet = await fastify.wallet.get(
+          request.params[options.walletId]
+        );
+        const content = request.body.content;
+        wallet.add(content);
+        await fastify.wallet.set(request.params[options.walletId], wallet);
+        reply.status(200).send({});
+      }
+    );
+    fastify.post(
```

Comment:
following REST conventions, this should be DELETE /:walletId/wallet/contents

Reply:
implemented.